src/js/common/components/NavButtons.jsx

```diff
@@ -81,7 +81,7 @@ export default class NavButtons extends React.Component {
     );
 
     let buttons;
-    if (path.endsWith('review-and-submit')) {
+    if (path.match(/review-and-submit(\/)?$/)) {
```

Comment:
Sounds good. In case you're interested in more like this: https://stackoverflow.com/questions/280634/endswith-in-javascript